### PR TITLE
Plugins 1185

### PR DESCRIPTION
**Pull request by hqiu-at-wiris**
Close button in popupmessage focused by default.

### CODE REVIEW

File: src/popupmessage.js [hunk 1]
```diff
@@ -109,8 +109,9 @@ export default class PopUpMessage {
       document.activeElement.blur();
 
       // For works with Safari.
-      window.focus();
+      // window.focus();
```
Comment (diegoWiris) [inline src/popupmessage.js]: Don't leave commented code.

**Comment by diegoWiris** [comment]
@HENRYQIULO You must make test

**Comment by hqiu-at-wiris** [comment]
@diegoWiris We already have tests for this issue, for example "closePopUpMessage" in 20modalwindow.xml, in another task I will create new tests for accessibility and tests like this will be moved.

**Review by diegoWiris — COMMENTED** [review]
> @diegoWiris We already have tests for this issue, for example "closePopUpMessage" in 20modalwindow.xml, in another task I will create new tests for accessibility and tests like this will be moved.

Nope. We don't have test to assert that cancel button is focused.

**Comment by hqiu-at-wiris** [comment]
> > @diegoWiris We already have tests for this issue, for example "closePopUpMessage" in 20modalwindow.xml, in another task I will create new tests for accessibility and tests like this will be moved.
> 
> Nope. We don't have test to assert that cancel button is focused.
I had updated also the plugins project, the test closeModalText in 20modalwindow.xml closes the popupmessage with ENTER whenever ESC is pressed, which means that Close button should be focused by default.

**Pull request merged**